internal/common/hashtool.go

```diff
@@ -37,7 +37,7 @@ func HashCommit(values []*big.Int, issig bool) *big.Int {
 	return new(big.Int).SetBytes(sha[:])
 }
 
-// GetHashNumber uses a hash to generate random numbers of a given bitlength in the fiat-shamir heuristic
+// GetHashNumber uses a hash to generate random numbers of a given bit-length in the fiat-shamir heuristic
```

Comment:
"bit length" would be better English here I think, as well as more consistent with the change "bitsize" -> "bit size" elsewhere.

Reply:
It looks a bit weird, I know, but I checked with [Wikipedia](https://en.wikipedia.org/wiki/Bit-length) (which also isn't flawless) and they state it as `bit-length`.